draft-ietf-mops-streaming-opcons.md

```diff
@@ -219,6 +288,46 @@ Presentations:
+
+The success of the largely TCP-based Internet is evidence that the mechanisms used to achieve equilibrium quickly, at a point where TCP senders do not interfere with other TCP senders for sustained periods of time, have been largely successful, even if the specific mechanisms used to reach equilibrium have change over time. Because TCP provides a common tool to avoid contention, new TCP-based applications (for example, HTTP) have had the same transport behavior as older TCP-based applications (for example, FTP). 
+
+In recent times, the TCP goal of probing for available bandwidth, and "backing off" when a network path is saturated, has been supplanted by the goal of avoiding growing queues along network paths, which prevent TCP senders from reacting quickly when a network path is saturated. Congestion control mechanisms such as COPA {{COPA}} and BBR {{I-D.cardwell-iccrg-bbr-congestion-control}} make these decisions based on measured path delays, assuming that if the measured path delay is increasing, the sender is injecting packets onto the network path faster than the receiver can accept them, so the sender should adjust its sending rate accordingly. 
+
+Although TCP protocol behavior has changed over time, the common practice of implementing TCP as part of an operating system kernel has acted to limit how quickly TCP behavior can change, even with the widespread use of automated operating system update installation on many end-user systems, so that streaming media providers could have a reasonable expectation that they could understand TCP transport protocol behaviors, and that those behaviors would remain relatively stable in the short term. 
+
+## The QUIC Protocol and QUIC Protocol Behavior
+
+The QUIC protocol {{I-D.ietf-quic-transport}}, developed from a proprietary protocol into an IETF standards-track protocol {{I-D.ietf-quic-transport}}, turns many of the statements made in {{udp-behavior}} and {{tcp-behavior}} on their heads. 
+
+The standardized QUIC protocol is used to carry HTTP traffic, but instead of being TCP-based, HTTP/3 {{I-D.ietf-quic-http}} is encapsulated in QUIC, which is then encapsulated in UDP, so streaming operators (and network operators) will see traffic that looks like HTTP, but is carried over UDP (specifically, using UDP port 443 {{Port443}}). Network operators may already be blocking UDP traffic on this port, since HTTP over TLS/SSL has used TCP, not UDP. Even if UDP traffic using this port is not blocked, traffic using this port may be severely rate-limited, since HTTP/3 over QUIC has the potential to send much more traffic over UDP than the network operator expects. 
```

Comment:
As someone overly familiar with HTTP/3 this paragraph reads a bit strangely. 

I might rephrase it to something like below (and defer to your editor capabilites to make it even better)

The QUIC transport protocol can carry different application-later protocols. HTTP/3 {{I-D.ietf-quic-http}} describes how QUIC transport features are used for HTTP. The convention is for HTTP/3 to run over UDP port 443 {{Port443}} but this is not a strict requirement. Since earlier versions of HTTP(S) rely on TCP, UDP ports may blocked for any values that are not commonly used, such as UDP 53 for DNS. Where UDP ports are not blocked and HTTP/3 can flow, streaming operators (and network operators) might see QUIC traffic patterns that are similar to HTTP(S) over TCP. Where ports are open, they may be servery rate-limited because operators may not be used to seeing legitimate high-bandwidth traffic such as streaming, over the ports that HTTP/3 tends to use.

Reply:
how about adding some links to the QUIC operability draft?

Reply:
I started with your text, although I jumped up and down on it a bit. And re: pointing to the manageability draft, duh, I am A Idiot for not thinking of that - thank you for the wake-up call!